haas/api.py

```diff
@@ -822,6 +813,9 @@ def show_node(nodename):
         'free': node.project_id is None,
         'nics': [{'label': n.label,
                   'macaddr': n.mac_addr,
+                  'networks': dict([(attachment.channel,
```

Comment:
So I played around with this and it impircally seems to work, but I have a strong objection to this: There's nothing in the json module's interface that says it respects the ordering, and it isn't required to --- json objects are unordered. We'd be relying on undocumented implementation details, and we should *never* do that.